en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc

```diff
@@ -334,6 +334,103 @@ When there are null values in custom expressions and are:
 * Fields without attributes, the null value is ignored. If the value in a required field, an error message occurs. If the value is in an optional field, the value is ignored and not sent to the connector. +
 * Fields with attributes, the value is ignored and not sent to the connector.
 
+== Define a Variable
+
+You can define variables in your Composer flow. Defining a variable allows you to complete certain use cases like aggregating calculations and storing values. Variables are defined locally within a flow and can be used anywhere within that flow. 
+
+[[access-the-variable-step]]
+=== Access the Variable Step 
+
+To access the *Variable* step: 
+
+. <<create-a-new-flow,Create a new flow>> or open an existing flow. +
+The *<Flow Name>* page appears.
+. Click *Edit*. 
+. Locate the position in your flow where you want access the *Variable* step and click the plus button. 
+. In the *Next Step* window, click *Variable*. +
+The *Variable* step appears. 
```

Comment:
Necessary?

Reply:
I think this one is necessary. This is to add a card to the flow that you can then interact with. I think this tells the user what they should see and sets up the subsequent steps that reference this set of steps.